flocker/apiclient/test/test_client.py

```diff
@@ -391,6 +473,28 @@ def test_this_node_uuid(self):
     return InterfaceTests
 
 
+def create_container_for_test(case, client, name=None):
+    """
+    :param TestCase case: The currently running test.
+    :param IFlockerClient client: The client for creating containers.
+    :param unicode name: The name to be assigned to the container or ``None``
+        to assign a random name.
```

Comment:
:grey_question: I don't know if we have a rule for what is required of docstrings... but this one is missing a summary and a returns section. I don't think the presence or absence of those docstrings will enhance or inhibit my personal development in future, so I'm indifferent, but I noticed they were missing.

I'll defer to your judgement on "what's normal" as you've worked in this codebase for longer.